openedx/features/offline_mode/apps.py

```diff
@@ -0,0 +1,14 @@
+"""
+OfflineMode application configuration
+"""
+
+
+from django.apps import AppConfig
+
+
+class OfflineModeConfig(AppConfig):
```

Comment:
Need to think if this app should become a plugin as other new edx-platform applications